### PR TITLE
fix: package.json run test integration in packages

### PR DESCRIPTION
**Pull request by adrien2p**
**What**
- fix product category repositories
- align tests
- fix categories routes
- fix package.json scripts

### CODE REVIEW

File: packages/modules/product/src/repositories/product-category.ts [hunk 1]
```diff
@@ -51,19 +51,27 @@ export class ProductCategoryRepository extends DALUtils.MikroOrmBaseTreeReposito
...
+      populate.includes("parent_category") ||
+      fields.some((field) => field.startsWith("parent_category"))
 
     if (shouldExpandParent) {
-      populate.indexOf("parent_category") === -1 &&
-        populate.push("parent_category")
+      // Since the tree is manually built, we dont want to get the relations
+      const relationIndex = populate.indexOf("parent_category")
+      relationIndex !== -1 && populate.splice(relationIndex, 1)
+      familyOptions.includeAncestorsTree ??= true
```
Comment (olivermrbl) [inline packages/modules/product/src/repositories/product-category.ts]: We cannot assume that wanting to expand `parent_category` is the same as wanting to include ancestors. 

`include_ancestors` will expand all parent categories, while as only passing `parent_category` should expand the immediate parent.

Comment (olivermrbl) [reply packages/modules/product/src/repositories/product-category.ts]: Same goes for descendants

Comment (adrien2p) [reply packages/modules/product/src/repositories/product-category.ts]: ok, got it, I ll change that 👍

**Review by carlos-r-l-rodrigues — APPROVED** [review]

**Comment by carlos-r-l-rodrigues** [comment]
@olivermrbl if you can have a look
this one is needed to unblock a couple other PRs.

**Review by olivermrbl — APPROVED** [review]

**Pull request merged**